joomla-extensions/jedchecker
# Add extra naming inspections to the INFO_XML rules

### Summary of Changes
Add extra naming inspections to the INFO_XML rules

New checks:
- check manifest file do exist
- check [naming rules](https://extensions.joomla.org/support/knowledgebase/for-jed-developers/extension-names/)

Code changes:
- fixed loading of the language file
- drop Joomla!1.5 support ("install" root element)

### Testing Instructions
Install, upload any extension (e.g. anyone with Joomla in title) and click "Check" button

### Actual result BEFORE applying this Pull Request
No such a checks are available at all.

### Expected result AFTER applying this Pull Request
Naming issue(s) found.

### Documentation Changes Required
None (as there is no official documentation page with detailed info on every rules)
